src/flow-var.h

```diff
@@ -33,6 +33,7 @@
 #define FLOWVAR_TYPE_STR 1
 #define FLOWVAR_TYPE_INT 2
 
+typedef uint8_t FlowVarKeyLenType;
```

Comment:
Why ?

Reply:
Abstraction; do you think it's excessive for something that's used in a handful of places?

Reply:
You do not use it in all the places you could then, right ?